pytest_fixtures/satellite_auth.py

```diff
@@ -93,8 +101,30 @@ def auth_source_ipa(module_org, module_loc, ipa_data):
     ).create()
 
 
+@pytest.fixture(scope='function')
+def auth_source_open_ldap(module_org, module_loc, open_ldap_data):
+    return entities.AuthSourceLDAP(
+        onthefly_register=True,
+        account=open_ldap_data['ldap_user_cn'],
+        account_password=open_ldap_data['ldap_user_passwd'],
+        base_dn=open_ldap_data['base_dn'],
+        groups_base=open_ldap_data['group_base_dn'],
+        attr_firstname=LDAP_ATTR['firstname'],
+        attr_lastname=LDAP_ATTR['surname'],
+        attr_login=LDAP_ATTR['login'],
+        server_type=LDAP_SERVER_TYPE['API']['posix'],
+        attr_mail=LDAP_ATTR['mail'],
+        name=gen_string('alpha'),
+        host=open_ldap_data['ldap_hostname'],
+        tls=False,
+        port='389',
+        organization=[module_org],
+        location=[module_loc],
+    ).create()
+
+
 @pytest.fixture
-def ldap_auth_source(request, module_org, module_location, ad_data, ipa_data):
+def ldap_auth_source(request, module_org, module_location, ad_data, ipa_data, open_ldap_data):
```

Comment:
It looks like every test function using `ldap_auth_source` is also using `ldap_tear_down` fixture too.  I would include it here, instead of the functions having to include it.  Enforce, through this fixture's use, that auth servers are torn down after the test using them is complete - at the scope that this fixture is configuring them in the first place.

Reply:
So for those test which use `ldap_auth_source`, we could for sure include `ldap_tear_down`there. Since some of the tests do not use `ldap_auth_source`  e.g `test_email_of_the_user_should_be_copied` which rely on the fixture `ldap_tear_down` which is in the top of the file, I rather make a new PR and modify there.

PS I tried moving the fixture from the test_ldap_authentication.py to pytest_fixtures/satellite_auth.py. Got index error and ergo I got irritated. :/